packages/instantsearch.js/src/widgets/index/index.ts

```diff
@@ -182,6 +195,25 @@ function getLocalWidgetsSearchParameters(
     }, initialSearchParameters);
 }
```

Comment:
I added it but only if `dependsOn` is `'search'` for now, there was still type inference problems when allowing for `undefined`, probably because we do `Partial` on `RequiredUiLifecycle` already...